### PR TITLE
[Enhance] add plane info in generated pkl file

### PR DESCRIPTION
**Pull request by ZCMax**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
This PR adds plane info in the generation process of pkl file. During the ablation study the ablation study of PointPillars on kitti, the ground plane infos is crucial for boosting the performance of models.  

## Modification
`tools/data_converter/kitti_data_infos.py` and `toools/data_converter/kitti_converter.py.`

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ZCMax** [comment]
Besides, the training datasets planes txt file should be provided, and the docs about kitti preparation  should be changed.

**Comment by ZCMax** [comment]
The docs has been modified, and the planes file has been provided.

File: tools/data_converter/kitti_converter.py [hunk 1]
```diff
@@ -113,6 +116,7 @@ def create_kitti_info_file(data_path,
         training=True,
         velodyne=True,
         calib=True,
+        plane=True,
```
Comment (Tai-Wang) [inline tools/data_converter/kitti_converter.py:119]: plane -> use_plane, also modify other arguments correspondingly

**Review by Tai-Wang — CHANGES_REQUESTED** [review]
Just need to fix comments

**Comment by ZCMax** [comment]
The plane infomations are generated by `Ground segmentation and occupancy grid generation using probability fields`, and the link of the code generating the planes infos has been dead.

File: tools/data_converter/kitti_data_utils.py [hunk 2]
```diff
@@ -251,6 +263,17 @@ def map_func(idx):
             calib_info['Tr_imu_to_velo'] = Tr_imu_to_velo
             info['calib'] = calib_info
 
+        if with_plane:
+            plane_path = get_plane_path(idx, path, training, relative_path)
+            if relative_path:
+                plane_path = str(root_path / plane_path)
+            with open(plane_path, 'r') as f:
```
Comment (ZwwWayne) [inline tools/data_converter/kitti_data_utils.py:270]: can we use lines = mmcv.list_from_file(plane_path)

**Comment by ZCMax** [comment]
@ZwwWayne

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**